<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QMainWindow" name="Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OCRLite</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/OCRLite.svg</normaloff>:/icons/OCRLite.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QComboBox" name="box_lang">
        <property name="toolTip">
         <string>语言</string>
        </property>
        <item>
         <property name="text">
          <string>中文</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/language.svg</normaloff>:/icons/language.svg</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>英文</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/language.svg</normaloff>:/icons/language.svg</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="box_engine">
        <property name="toolTip">
         <string>引擎</string>
        </property>
        <item>
         <property name="text">
          <string>百度</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/baidu.svg</normaloff>:/icons/baidu.svg</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>微软</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/tex.svg</normaloff>:/icons/tex.svg</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tesseract</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/google.svg</normaloff>:/icons/google.svg</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="box_sort">
        <property name="toolTip">
         <string>整理</string>
        </property>
        <item>
         <property name="text">
          <string>复制时</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/sort.svg</normaloff>:/icons/sort.svg</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>即时</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/sort.svg</normaloff>:/icons/sort.svg</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>永不</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/sort.svg</normaloff>:/icons/sort.svg</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="box_screen">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>屏幕</string>
        </property>
        <item>
         <property name="text">
          <string>屏幕0</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/screen.svg</normaloff>:/icons/screen.svg</iconset>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="QLabel" name="image_viewer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>图片拖这里</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextEdit" name="text_editor"/>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_copy">
        <property name="toolTip">
         <string>复制</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/copy.svg</normaloff>:/icons/copy.svg</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btn_screenshot">
        <property name="toolTip">
         <string>截图</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/screenshot.svg</normaloff>:/icons/screenshot.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_setting">
        <property name="toolTip">
         <string>设置</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/setting.svg</normaloff>:/icons/setting.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_append">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>追加</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
